### DISCUSSION TITLE
希望新版加入漫画章节更新日期显示

### DISCUSSION THREAD

**Discussion by abcd79687** [xiaoyaocz/flutter_dmzj#110] [General]
![Screenshot_2023-03-15-18-38-25-252_com nsapp dmzj](https://user-images.githubusercontent.com/127945022/225285507-0430003f-b90c-4381-b2aa-fbd05f61c1ef.jpg)
就是旧版这种漫画章节列表后附带具体更新日期的。

**Comment by xiaoyaocz** [comment]
#112
